packages/components/autocomplete/src/autocomplete.vue

```diff
@@ -402,5 +402,7 @@ defineExpose({
   close,
   /** @description highlight an item in a suggestion */
   highlight,
+  /** @description run load results */
```

Comment:
The description here could perhaps be tweaked. `loading suggestion list`

Reply:
Done